CMakeLists.txt

```diff
@@ -2312,6 +2322,18 @@ endif()
 
 if(HPX_WITH_EXAMPLES)
   add_hpx_pseudo_target(examples)
+  if(HPX_WITH_FETCH_HWLOC AND "${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+    add_custom_target(
+      HwlocDLL ALL
+      COMMAND ${CMAKE_COMMAND} -E make_directory
+              "${CMAKE_BINARY_DIR}/$<CONFIG>/bin/"
+      COMMAND
+        ${CMAKE_COMMAND} -E copy_if_different
+        "${HWLOC_ROOT}/bin/libhwloc-15.dll"
+        "${CMAKE_BINARY_DIR}/$<CONFIG>/bin/"
+    )
+    add_hpx_pseudo_dependencies(examples HwlocDLL)
+  endif()
 endif()
```

Comment:
@vrnimje what happens when `HPX_WITH_EXAMPLES` is not set? Please make a new PR if this is incorrect

Reply:
I could only find `examples` as a psuedo-target, to which I added the dependency which I defined. I was not able to find any other target, which is built by default
Is there any other mechanism through which this can be achieved?

Reply:
I was only able to find `examples` as a psuedo-target, to which I could add the dependency I have defined (`HwlocDLL`)
Since the libhwloc-15.dll needs to be copied exactly to `"${CMAKE_BINARY_DIR}/$<CONFIG>/bin/"`, and the Configuration type `$<CONFIG>` is defined during the build stage, I used the above method. As far as I understand, `$<CONFIG>` can only be obtained using `add_custom_command` or `add_custom_target`.

Is there any other mechanism to achieve this?

Reply:
I now realise that I can directly add `HwlocDLL` as a pseudo target using `add_hpx_pseudo_target`